pkg/plugins/jwe/api.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jwe
+
+import "github.com/google/tink/go/keyset"
+
+// Provider interface for Crypto ctx
+type Provider interface {
+}
+
+// Creator method to create new pluggable JWE service
+type Creator func(prov Provider) (AriesJWE, error)
+
+// AriesJWE is an interface defining a pluggable JWE builder operations provider to Aries agents.
+// It's responsible for building JWE structures for either 1: sealing them or 2: open serialized ones. It depends on
+// on the AriesCrypto to encrypt/decrypt content and AriesKMS to get key handles needed for crypto operations.
+type AriesJWE interface {
```

Comment:
Same comment as earlier - We shouldn't add prefixes.

Reply:
will remove

Reply:
prefix removed